README.md

```diff
@@ -416,7 +416,7 @@ NaN values.
 * url: the observe url to query the data from.
 
 
-* user: the observe user for querying the data.
+* user: the first part of the url before the `.` (this parameter is optional, since it is infered usually). This is not the ID of your personal account. You don't have to specify that since you use an access token. 
```

Comment:
should be the exact same definition as above; also needs to be changed in the code as the readme is largely auto-generated

Reply:
Done